vms/avm/states/state.go

```diff
@@ -576,3 +567,55 @@ func (s *state) writeStatuses() error {
 	}
 	return nil
 }
+
+func (s *state) CleanupTxs() error {
```

Comment:
Before merging we should make sure that this doesn't take too long on mainnet. <2m should be good. More than that - we need to think about it. >5m we probably need to perform this async.

Reply:
This takes >5m on mainnet.